docs/charts/line.md

```diff
@@ -79,6 +79,11 @@ The style of point. Options are:
 
 If the option is an image, that image is drawn on the canvas using [drawImage](https://developer.mozilla.org/en/docs/Web/API/CanvasRenderingContext2D/drawImage).
 
+### stepInterpolation
```

Comment:
We still need `false` and we use `true` as shorthand pretty often so I would not deprecate boolean values.